packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -275,23 +294,64 @@ function FontCollection( { slug } ) {
 			) }
 
 			{ ! renderConfirmDialog && ! selectedFont && (
-				<FontsGrid>
-					{ fonts.map( ( font ) => (
-						<FontCard
-							key={ font.font_family_settings.slug }
-							font={ font.font_family_settings }
-							onClick={ () => {
-								setSelectedFont( font.font_family_settings );
-							} }
-						/>
-					) ) }
-				</FontsGrid>
+				<VStack spacing={ 0 }>
+					<div className="font-library-modal__fonts-grid__main">
+						{ items.map( ( font ) => (
+							<FontCard
+								key={ font.font_family_settings.slug }
+								font={ font.font_family_settings }
+								onClick={ () => {
+									setSelectedFont(
+										font.font_family_settings
+									);
+								} }
+							/>
+						) ) }
+					</div>
+				</VStack>
 			) }
 		</TabPanelLayout>
 	);
 }
 
-function Footer( { handleInstall, isDisabled } ) {
+function PaginationFooter( { page, totalPages, setPage } ) {
+	return (
+		<Flex justify="center">
+			<Button onClick={ () => setPage( 1 ) } disabled={ page === 1 }>
+				&lt;&lt;
+			</Button>
+			<Button
+				onClick={ () => setPage( page - 1 ) }
+				disabled={ page === 1 }
+			>
+				&lt;
+			</Button>
+			<Text>{ `Page ` }</Text>
+			<SelectControl
+				value={ page }
+				options={ [ ...Array( totalPages ) ].map( ( e, i ) => {
+					return { label: i + 1, value: i + 1 };
+				} ) }
+				onChange={ ( newPage ) => setPage( parseInt( newPage ) ) }
+			/>
+			<Text>{ ` of ${ totalPages }` }</Text>
```

Comment:
The string `Page of X` cannot be translated properly. You may be able to use [this approach](https://github.com/WordPress/gutenberg/blob/7d28fa67ef194947afb3c5b1fa08798b8d8fa77d/packages/dataviews/src/pagination.js#L34).

Reply:
The information that you are page X of X is not announced the screen reader users.

The selectControl itself is missing a label, so there is no description of what this control does.

Reply:
I don't have a problem with accessibility, I have a problem with localization. Because the translation function has not been applied to "Page" and "of X" and their respective texts have been split, their order is fixed.

Reply:
Thank you @t-hamano I have copied that approach and I hope that what is available now will work as desired.

Addressed in [this commit](https://github.com/WordPress/gutenberg/pull/58794/commits/a5bf6a1cc154bec8f14a638c870b4c80c419eeb5).